src/player_display.cpp

```diff
@@ -1326,27 +1327,33 @@ void player::disp_info()
     effect_win_size_y--;
 
     // Print name and header
-    // Post-humanity trumps your pre-Cataclysm life.
-    if( crossed_threshold() ) {
-        std::string race;
-        for( const trait_id &mut : get_mutations() ) {
-            const mutation_branch &mdata = mut.obj();
-            if( mdata.threshold ) {
-                race = mdata.name();
-                break;
+    // Post-humanity trumps your pre-Cataclysm life
+    // Unless you have a custom profession.
+    std::string race;
+    if( g->u.custom_profession.empty() || g->u.custom_profession == "" ) {
```

Comment:
These are the same check.
```suggestion
    if( g->u.custom_profession.empty() ) {
```